Sources/Routing/Protocols/NavigationPathManageable.swift

```diff
@@ -10,28 +10,27 @@ import SwiftUI
 public protocol NavigationPathManageable: ObservableObject {
     associatedtype Destination: ViewDisplayable
 
-    var path: NavigationPath { get set }
+    var path: [Destination] { get set }
 
-    func pop(_ count: Int)
+    func popTo(_ destination: Destination)
```

Comment:
```suggestion
    func pop(to destination: Destination)
```

Reply:
I think it'd still be useful to have a function for `func pop(by count: Int)`

Reply:
I returned it back